training_config/multilang_dependency_parser.jsonnet

```diff
@@ -0,0 +1,120 @@
+            "fr",
+            "pt",
+            "sv"
+        ],
+        "pos_tag_embedding": {
+            "embedding_dim": 50,
+            "vocab_namespace": "pos"
+        },
+        "tag_representation_dim": 100,
+        "text_field_embedder": {
+            "token_embedders": {
+                "elmo": {
+                    "type": "elmo_token_embedder_multilang",
+                    "aligning_files": {
+                        "en": "https://www.dropbox.com/s/nufj4pxxgv5838r/en_best_mapping.pth?dl=1",
+                        "es": "https://www.dropbox.com/s/6kqot8ssy66d5u0/es_best_mapping.pth?dl=1",
+                        "fr": "https://www.dropbox.com/s/0zdlanjhajlgflm/fr_best_mapping.pth?dl=1",
+                        "it": "https://www.dropbox.com/s/gg985snnhajhm5i/it_best_mapping.pth?dl=1",
+                        "pt": "https://www.dropbox.com/s/skdfz6zfud24iup/pt_best_mapping.pth?dl=1",
+                        "sv": "https://www.dropbox.com/s/o7v64hciyifvs8k/sv_best_mapping.pth?dl=1",
+                        "de": "https://www.dropbox.com/s/u9cg19o81lpm0h0/de_best_mapping.pth?dl=1"
+                    },
+                    "do_layer_norm": false,
+                    "dropout": 0.3,
+                    "scalar_mix_parameters": [
+                        -9e10,
+                        1,
+                        -9e10
+                    ],
+                    "options_files": {
+                        "en": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "es": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "fr": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "it": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "pt": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "sv": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1",
+                        "de": "https://www.dropbox.com/s/ypjuzlf7kj957g3/options262.json?dl=1"
+                    },
+                    "weight_files": {
+                        "en": "https://www.dropbox.com/s/1h62kc1qdcuyy2u/en_weights.hdf5?dl=1",
+                        "es": "https://www.dropbox.com/s/ygfjm7zmufl5gu2/es_weights.hdf5?dl=1",
+                        "fr": "https://www.dropbox.com/s/mm64goxb8wbawhj/fr_weights.hdf5?dl=1",
+                        "it": "https://www.dropbox.com/s/owfou7coi04dyxf/it_weights.hdf5?dl=1",
+                        "pt": "https://www.dropbox.com/s/ul82jsal1khfw5b/pt_weights.hdf5?dl=1",
+                        "sv": "https://www.dropbox.com/s/boptz21zrs4h3nw/sv_weights.hdf5?dl=1",
+                        "de": "https://www.dropbox.com/s/2kbjnvb12htgqk8/de_weights.hdf5?dl=1"
+                    }
+                }
+            }
+        }
+    },
+    "train_data_path": std.extVar("TRAIN_PATHNAME"),
```

Comment:
Is the data freely distributable?  If it is, can we put it on S3 also?

Reply:
The data for all languages except for English is freely available here - https://github.com/ryanmcd/uni-dep-tb.
The English trees are based on PTB, so like the biaffine parser, we can't provide it. But we can provide a pre-trained model and maybe give in the default configuration file all other languages with a comment that it won't reproduce the results from the paper.
Is there currently a way to put a url of a tarball in the configuration that will be extracted in a way that will allow us to keep the multiple training files logic?

Reply:
Hmm, I don't think there's a way of giving such a tarball.  Maybe we should just leave a comment here describing how to get the data and set it up correctly (or point to the dataset reader docstring, if that's the right place to have this info).